### PR TITLE
PT19 - Changes to 3.0.0

### PR DESCRIPTION
**Pull request by RohdeBSH**
Changes to edition 3.0.0 as requested by PT19.

### CODE REVIEW

**Comment by RohdeBSH** [comment]
The FC is currently valid. It is not mandatory to specify the TAG “definitionReference”. Nevertheless, it is of course not a good option to omit the reference for the “iD” attribute to the IHO registry. This must be corrected in any case after approval for this attribute.

The BSH is currently in contact with the KHOA to check the possibility of obtaining the value for the “sourceIdentifier” TAG prior to approval.

**Comment by RohdeBSH** [comment]
The FC remains as it is, as it is not possible to determine the "sourceIdentifier" TAG before the approval time has expired.

**Comment by DavidGrant-NIWC** [comment]
> According to S-100 Ed. 5.2.0 clause 17-4.2 Figure 17-3 & Figure 17-4 the file names of the FeatureCatalogues only starts with the product specification number without “S-“.

* There is no issue with renaming the catalog but note that the cited figure is just one example.
  * The example is only applicable to catalogs distributed via an exchange set. 
  * Exchange set producers may rename content as they see fit.
  * Note that the name of the catalog is specified in CATALOG.XML; applications should not rely on any particular naming convention.
* In general, there is no catalog naming convention that I'm aware of although the PS or IHO policy could dictate one.

**Review by AnnaWall01 — COMMENTED** [review]
Thank you for all the work Daniel. Regarding this point (Root-Bounding Box), I asked Lynn for a good wording for the Comment Sheet. She suggested to me the following: "...bounding box needs to encompass all data. Including no fill values." Is it possible squeeze this sentence in somehow? Or if you feel that it's already the same as you have already written, then that is fine.

I know you asked for help regarding this one earlier, sorry I didn't add this then...

**Review by DavidGrant-NIWC — APPROVED** [review]
* Looks good but note previous comment on rename of FC. 
* Putting the version number in the filename is asking for problems.
  * requires renaming the file via Git on every version change.
  * users unfamiliar with Git may rename on their desktop which can cause a delete + add operation losing file history.

**Comment by RohdeBSH** [comment]
> > According to S-100 Ed. 5.2.0 clause 17-4.2 Figure 17-3 & Figure 17-4 the file names of the FeatureCatalogues only starts with the product specification number without “S-“.
> 
>     * There is no issue with renaming the catalog but note that the cited figure is just one example.
>       
>       * The example is only applicable to catalogs distributed via an exchange set.
>       * Exchange set producers may rename content as they see fit.
>       * Note that the name of the catalog is specified in CATALOG.XML; applications should not rely on any particular naming convention.
> 
>     * In general, there is no catalog naming convention that I'm aware of although the PS or IHO policy could dictate one.

Hi @DavidGrant-NIWC ,

I know it's just an example and there is no naming convention.
But I don't think it's correct and appropriate just because it's possible to deviate from the example. In my opinion, that only creates confusion. That's why I think it's correct for us to take this very simple route to avoid any questions arising in the first place.
It doesn't cost us anything.
I would like to see a naming convention that is prescribed by the IHO or S-100.

**Review by AnnaWall01 — APPROVED** [review]
Thanks for adding that @RohdeBSH. I have nothing else to comment on. The changes looks good.

File: sources/3.0.0/sections/10-data_product_format.adoc [hunk 1]
```diff
@@ -232,6 +232,8 @@ Example: INT.IHO.S-102.3.0.0
 .4+| The values are in decimal degrees. +
 If a projected CRS is used for the dataset, these values refer to those of the baseCRS underlying the projected CRS (see <<note3>>).
 
+The root bounding box needs to encompass all data. Including no fill values. The outermost cell boundaries of the grid cells and the bounding box / domainExtent.polygon of each feature instance group form the basis for the root bounding box.
```
Comment (giumas) [inline sources/3.0.0/sections/10-data_product_format.adoc:235]: "The root bounding box needs to encompass all data, including no fill values." to clarify that the inclusion of the no fill values is referred to "all data".

Comment (giumas) [reply sources/3.0.0/sections/10-data_product_format.adoc:235]: "The bounding box and the domain extent polygon of each feature instance group form the basis for the root bounding box." because the previous sentence already takes care of the cells.

**Review by giumas — APPROVED** [review]
A couple of proposed changes to clarify/simplify the wording.

File: sources/3.0.0/sections/10-data_product_format.adoc (hunk 1 above)
Comment (RohdeBSH) [reply sources/3.0.0/sections/10-data_product_format.adoc:235]: > "The bounding box and the domain extent polygon of each feature instance group form the basis for the root bounding box." because the previous sentence already takes care of the cells.

I wanted to mention the outer cell boundaries because they are also explicitly mentioned in other sections of the PS. Is it ok for you if I leave this part as it is?

Comment (giumas) [reply sources/3.0.0/sections/10-data_product_format.adoc:235]: That's fine. Can we at least use "domain extent polygon" in place of "domainExtent.polygon"? Mainly for being aligned with "the bounding box".
